v1/python-sdk/tutorials/automl-with-azureml/image-instance-segmentation/auto-ml-image-instance-segmentation.ipynb

```diff
@@ -435,7 +435,7 @@
     "}\n",
     "\n",
     "tuning_settings = {\n",
-    "    \"iterations\": 10,\n",
+    "    \"iterations\": 6,\n",
```

Comment:
Assuming you saw a successful run with 6 iterations.

Reply:
Yes. In my local